src/main/java/com/exercise/sfe/entity/GiftCertificateToTag.java

```diff
@@ -0,0 +1,23 @@
+package com.exercise.sfe.entity;
+
+import lombok.*;
+
+import java.io.Serializable;
+
+@Getter @Setter
```

Comment:
Please, never place 2 annotations at the same line. And investigate Java Code Conventions